web/packages/core/src/ruffle-player.ts

```diff
@@ -511,11 +511,20 @@ export class RufflePlayer extends HTMLElement {
             this.loadedConfig
         );
         this._cachedDebugInfo = this.instance!.renderer_debug_info();
-        console.log(
-            "New Ruffle instance created (WebAssembly extensions: " +
-                (ruffleConstructor.is_wasm_simd_used() ? "ON" : "OFF") +
-                ")"
-        );
+        {
```

Comment:
Is there a reason for opening an inner scope?

Reply:
None really, apart from ensuring that the variable name `actuallyUsedRendererName` is not accessible later in the outer scope - I can remove it if you want.

Reply:
Yeah, removing it would be nicer in my opinion.